plugins/woocommerce/includes/admin/meta-boxes/views/html-product-data-inventory.php

```diff
@@ -10,7 +10,11 @@
 }
 ?>
 <div id="inventory_product_data" class="panel woocommerce_options_panel hidden">
-
+	<div class="inline notice woocommerce-message show_if_variable">
+		<p>
+			<?php echo esc_html_e( 'Settings below apply to all variations without manual stock management enabled. ', 'woocommerce' ); ?> <a target="_blank" href="https://woocommerce.com/document/variable-product/"><?php esc_html_e( 'Learn more', 'woocommerce' ); ?></a>
+		</p>
+	</div>
```

Comment:
We can abstract this in a follow-up PR. We might create a method like `woocommerce_wp_notice` or `woocommerce_wp_message` [here](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/includes/admin/wc-meta-box-functions.php).